Online/Menus/CSS/HandleInputsOnCSS.asm

```diff
@@ -157,12 +159,28 @@ b SKIP_START_MATCH
 ################################################################################
 HANDLE_CONNECTED:
 
-# Handle disconnect
-rlwinm.	r0, REG_INPUTS, 0, 0x10
-beq SKIP_DISCONNECT
+# Handle disconnect when input is hold for X seconds
+branchl r12, Inputs_GetPlayerHeldInputs
+li r3, 0x4
```

Comment:
no mask here, should use

rlwinm. r0,r4,0,0x10

Reply:
Sure I will do the change, however I don't really understand what you mean with the mask, could you explain please? (I kind of get confused with rotate instructions)

Reply:
Oh, nvm I kind of got it now lol.